code/final/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <link rel="icon" href="icon.png">
+  <title>Hire a developer!</title>
+</head>
+
+<body>
+
+  <header class="v-header header-container">
+    <div class="fullscreen-video-wrap">
+      <video id="myvideo" preload autoplay muted loop playsinline>
+        <!-- source tag goes here with javascript -->
```

Comment:
You can delete this comment! It's just a helper text for when starting the project ☺️
```suggestion
```